components/competition-schedule.tsx

```diff
@@ -0,0 +1,62 @@
+import fs from 'fs';
+import path from 'path';
+
+interface Competition {
+  id: string;
+  name: string;
+  status: string;
+  submissionDeadline: string;
+  resultsDate: string;
+  url: string | null;
+}
+
+interface CompetitionScheduleProps {
+  className?: string;
+}
+
+export async function CompetitionSchedule({ className }: CompetitionScheduleProps) {
```

Comment:
Based on discord discussions, let's just leave this, but comment it out on the page for now?